tests/Zend/Loader/AutoloaderFactoryTest.php

```diff
@@ -30,6 +30,12 @@
 require_once 'Zend/Loader/AutoloaderFactory.php';
 require_once 'Zend/Loader/ClassMapAutoloader.php';
 require_once 'Zend/Loader/StandardAutoloader.php';
+// Trigger autoloader for these
+class_exists('PHPUnit_Framework_Constraint_IsNull');
+class_exists('PHPUnit_Framework_Constraint_IsTrue');
+class_exists('SebastianBergmann\Exporter\Exporter');
+class_exists('PHPUnit_Framework_Constraint_IsEqual');
+class_exists('PHPUnit_Framework_Constraint_Exception');
```

Comment:
When calling this test file directly, these classes aren't available. When ran as part of the Zend_Loader test suite, it was working, because the assertions were being called beforehand. The the test that I removed below removed one of those necessary assertions (assertNotNull), so it was failing even when running as part of the suite.